rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/ContextMenu.java

```diff
@@ -150,7 +154,15 @@ public static Menu createContextMenuFromCompactView(RMController controller, Obj
             menu.disableItems(deployItem, undeployItem, editItem);
         }
 
-        menu.setItems(deployItem, undeployItem, editItem, new MenuItemSeparator(), lockItem, unlockItem, removeItem);
+        menu.setItems(deployItem,
+                      undeployItem,
+                      editItem,
+                      new MenuItemSeparator(),
+                      lockItem,
+                      unlockItem,
+                      removeItem,
+                      new MenuItemSeparator(),
+                      exportItem);
 
         return menu;
     }
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)